.github/workflows/kvcounter-as-release.yml

```diff
@@ -0,0 +1,53 @@
+name: KVCOUNTER-AS-RELEASE
+
+on:
+  push:
+    tags:
+    - 'kvcounter-as-v*'
+
+env:
+  CARGO_TERM_COLOR: always
+  working-directory: ./kvcounter-as
+  DESTINATION: build/kvcounter-as_s.wasm
+  WASH_ISSUER_KEY: ${{ secrets.WASMCLOUD_ACCOUNT_OFFICIAL }}
+  WASH_SUBJECT_KEY: ${{ secrets.WASMCLOUD_KVCOUNTER }}
+  WASH_REG_USER:  ${{ secrets.AZURECR_PUSH_USER }}
+  WASH_REG_PASSWORD:  ${{ secrets.AZURECR_PUSH_PASSWORD }}
+  REVISION: ${{ github.run_number }}
+
+jobs:
+  github_release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Create Release
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.ref }}
+        release_name: Release ${{ github.ref }}
+        draft: false
+        prerelease: true
+
+  artifact_release:
+    needs: github_release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          curl -s https://packagecloud.io/install/repositories/wasmCloud/core/script.deb.sh | sudo bash
+          sudo apt install wash
+      # Push artifact to https://AZURECR/httpclient:VERSION
+      - name: npm install
+        run: npm install
+      - name: npm build
+        run: npm run build
+      - name: Wash Sign Claim
+        run: wash claims sign build/kvcounter-as.wasm --issuer ${{ secrets.WASM_ACCOUNT_OFFICIAL }} --subject ${{ secrets.WASMCLOUD_KVCOUNTER_AS }} -c wasmcloud:keyvalue -c wasmcloud:httpserver --name "Key Value Counter"
```

Comment:
I know GH actions is smart enough to remove the secret from the command line here, but since you have `WASH_ISSUER_KEY` and `WASH_SUBJECT_KEY` set in the environment you won't need to use the `--issuer` or `--subject` flag here. You can add the `--disable-keygen` flag to ensure that this will fail if it doesn't work.

Also, here we should supply the `--ver` flag with the actor version, and the `--rev` flag with the revision number